README.md

```diff
@@ -145,73 +145,64 @@ in the Minishift docker environment, as detailed in next section.
 dev-scripts/docker_create_images_and_tag_in_minishift.sh
 ```
 
-## How to deploy or roll-update it in Minishift
+## How to deploy or roll-update it in Minishift or OSIO
     
 ### Deployment prerequisites
 
-* Get [minishift](https://github.com/minishift/minishift#installation) (we have tested with v1.0.0beta4)
-  * Minishift v1.0.0beta4 includes the `oc` binary, and this should be added to your `$PATH`. The binary is located
-    in the `.minishift/cache/oc/<oc version>/` directory (e.g. on Linux, it is in `~/.minishift/cache/oc/v1.4.1/`).
-* Get [gofabric8](https://github.com/fabric8io/gofabric8#getting-started) (we have tested with v0.4.121)
-* Clone fabric8-online git repository:
+* [minishift](https://github.com/minishift/minishift#installation) (v1.0.0 or greater)
+* `oc` the OpenShift command line client tool. The binary is included in minishift distribution and can be found in folder `.minishift/cache/oc/<oc version>/`. You can add this folder to your `$PATH` using the following command `eval $(minishift oc-env)`.
 
-`git clone https://github.com/fabric8io/fabric8-online.git`
-
-* The scripts assume that the `fabric8-online` repository is cloned in `${HOME}/github/fabric8-online/`.
-If you cloned it in another location, you need to override the `FABRIC8_ONLINE_PATH` variable
-with the full path of the cloned `fabric8-online` git repository :
+### Deploy to Minishift 
 
 ```bash
-export FABRIC8_ONLINE_PATH="<path where you cloned it>/fabric8-online/"
+SCRIPT_URL=https://raw.githubusercontent.com/redhat-developer/rh-che/master/dev-scripts/openshift_deploy.sh
+curl -fsSL ${SCRIPT_URL} -o get-che.sh && sh get-che.sh
 ```
-__Important note:__ the `FABRIC8_ONLINE_PATH` should always end with a `/`.
 
-* For all the scripts that deploy or rollout to Minishift, you should have minishift running.
-This can be done with the following command:
+### Deploy to openshift.io 
 
 ```bash
-minishift start
+SCRIPT_URL=https://raw.githubusercontent.com/redhat-developer/rh-che/master/dev-scripts/openshift_deploy.sh
```

Comment:
why do we need to download the deployment script that we have locally ?
why not:
```
export OPENSHIFT_FLAVOR=osio && dev-scripts/openshift-deploy.sh
```
at least in the readme ?

The download variant is interesting when not having the `rh-che` gitHub repo locally, but it's not the main case IMO.